packages/dataviews-client/src/dataviews-client.ts

```diff
@@ -1,7 +1,7 @@
 import template from 'lodash/template'
 import { stringify } from 'qs'
 import GFWAPI, { FetchResponseTypes, FetchOptions } from '@globalfishingwatch/api-client'
-import { Dataview, Resource, WorkspaceDataviewConfig } from './types'
+import { Dataview, Resource, DataviewInstance } from './types'
```

Comment:
I think you modified that just doing a search & replace, but I think we could get rid of this file and think about where to put `resolveEndpoint` and the types.

Reply:
Good idea, I'll move in another PR this and some other several types to a new package to include only types